inc/namespace.php

```diff
@@ -89,9 +89,19 @@ function load_instant_articles() {
  * @return string
  */
 function robots_txt( string $output ) : string {
-	if ( file_exists( ROOT_DIR . '/config/robots.txt' ) ) {
+	// If the legacy-style file exists, load it, but warn.
+	// Legacy file will be in the `/config` dir instead of `/.config`.
+	$legacy_file = ROOT_DIR . '/config/robots.txt';
+	if ( file_exists( $legacy_file ) ) {
+		trigger_error( 'The "config/robots.txt" file is deprecated as of Altis 2.0. Use ".config/robots.txt" instead.', E_USER_DEPRECATED );
+
+		// @codingStandardsIgnoreLine
+		$output .= "\n" . file_get_contents( $legacy_file ) . "\n";
```

Comment:
So as you say rather than concatenating both files if they exist we should prioritise the one at `.config/robots.txt` instead if it exists.